### PR TITLE
[zh] sync/administrator/monitoring/working-with-filebeat.md

### PR DESCRIPTION
**Pull request by liangyongzhenya**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/monitoring/working-with-filebeat.md [hunk 1]
```diff
@@ -172,71 +172,78 @@ hack/local-up-karmada.sh
...
-         - name: data
-           hostPath:
-             path: /var/lib/filebeat-data
-             type: DirectoryOrCreate
-   ```
-
-2. Run the below command to execute Karmada PropagationPolicy and ClusterPropagationPolicy. 
-
+         - name: varlibdockercontainers
+        hostPath:
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/monitoring/working-with-filebeat.md:176]: Line 176-184 repeated.

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/monitoring/working-with-filebeat.md [hunk 2]
```diff
@@ -172,71 +172,78 @@ hack/local-up-karmada.sh
...
    ```
+2. 运行下面的命令来执行 Karmada PropagationPolicy 和 ClusterPropagationPolicy。
+   ```yaml
    cat <<EOF | kubectl apply -f -
    apiVersion: policy.karmada.io/v1alpha1
    kind: PropagationPolicy
    metadata:
-     name: filebeat-propagation
-     namespace: logging
+   name: filebeat-propagation
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/monitoring/working-with-filebeat.md:197]: This yaml file is not indented.

**Review by Poor12 — COMMENTED** [review]
Great thanks for translations!!

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/monitoring/working-with-filebeat.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
 ---
-title: Use Filebeat to collect logs of Karmada member clusters
+title: 使用 Filebeat 来收集 Karmada 成员集群的日志
 ---
 
-[Filebeat](https://github.com/elastic/beats/tree/master/filebeat) is a lightweight shipper for forwarding and centralizing log data. Installed as an agent on your servers, Filebeat monitors the log files or locations that you specify, collects log events, and forwards them either to [Elasticsearch](https://www.elastic.co/products/elasticsearch) or [kafka](https://github.com/apache/kafka) for indexing. 
+[Filebeat] (https://github.com/elastic/beats/tree/master/filebeat) 是一个轻量级的运送器，用于转发和集中日志数据。作为代理安装在你的服务器上，Filebeat 监控你指定的日志文件或位置，收集日志事件，并将它们转发到 [Elasticsearch] (https://www.elastic.co/products/elasticsearch) 或 [kafka] (https://github.com/apache/kafka) 以进行索引。
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/monitoring/working-with-filebeat.md:5]: ```[Filebeat](https://github.com/elastic/beats/tree/master/filebeat)```
```[Elasticsearch](https://www.elastic.co/products/elasticsearch)```

**Review by Poor12 — COMMENTED** [review]
BTW, you can see https://karmada.io/docs/next/contributor/contribute-docs for how to preview docs. :)

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/monitoring/working-with-filebeat.md (hunk 3 above)
Comment (liangyongzhenya) [reply i18n/zh/docusaurus-plugin-content-docs/current/administrator/monitoring/working-with-filebeat.md:5]: Thank you very much！

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/monitoring/working-with-filebeat.md [hunk 4]
```diff
@@ -234,9 +233,8 @@ hack/local-up-karmada.sh
    EOF
    ```
 
-3. Obtain the collected logs according to the `output` configuration of the `filebeat.yml`.
-
-## Reference
+3. 根据 filebeat.yml 的 out 配置，获取收集的日志。
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/monitoring/working-with-filebeat.md:236]: output 配置

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**